backend/app/api/journal.py

```
@@ -105,6 +113,179 @@ async def parse_stt(
+    if not (file.content_type or "").startswith("image/"):
+        raise HTTPException(400, "이미지 파일만 업로드 가능합니다.")
+    data = await file.read()
+    if not data:
+        raise HTTPException(400, "빈 이미지 파일입니다.")
+    if len(data) > settings.JOURNAL_VISION_MAX_BYTES:
+        limit_mb = settings.JOURNAL_VISION_MAX_BYTES // (1024 * 1024)
+        raise HTTPException(413, f"사진 크기는 {limit_mb}MB 이하여야 합니다.")
+
+    try:
+        meta = save_photo(current_user.id, data)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)